plugins/ingest/src/main/resources/org/elasticsearch/ingest/processor/grok/pattern.properties

```diff
@@ -0,0 +1,17 @@
+aws=aws
```

Comment:
yes, I ported most of the tests from ruby-grok and logstash. we don't have many of these tests there :(